client/src/actions/index.ts

```diff
@@ -251,6 +252,18 @@ const dispatchDiffExpErrors = (
   }
 };
 
+const genesetsFetch = (dispatch: AppDispatch) => {
+  const defaultResponse = {
```

Comment:
let's extract this out as a constant, so we don't create a new object every time

e.g.,

```tsx
const DEFAULT_RESPONSE = { ... }


```